### PR TITLE
bootstrap: implement custom buckets

### PR DESCRIPTION
**Pull request by kyessenov**
Implements https://github.com/istio/api/pull/2800

### CODE REVIEW

File: pkg/bootstrap/config.go [hunk 1]
```diff
@@ -251,13 +252,28 @@ func getStatsOptions(meta *model.BootstrapNodeMetadata) []option.Instance {
 		inclusionSuffixes = requiredEnvoyStatsMatcherInclusionSuffixes
 	}
 
+	var buckets []option.HistogramBucket
+	if bucketsAnno, ok := meta.Annotations[annotation.SidecarStatsHistogramBuckets.Name]; ok {
+		js := map[string][]float64{}
+		err := json.Unmarshal([]byte(bucketsAnno), &js)
+		if err == nil {
```
Comment (howardjohn) [inline pkg/bootstrap/config.go:259]: nit: log the error if its invalid maybe?

**Review by lei-tang — APPROVED** [review]

**Review by howardjohn — COMMENTED** [review]
looks good

**Review by howardjohn — APPROVED** [review]

File: pkg/bootstrap/config.go (hunk 1 above)
Comment (kyessenov) [reply pkg/bootstrap/config.go:259]: done

**Review by zirain — APPROVED** [review]
LGTM, can you add release-notes?

**Pull request merged**